cuebot/src/test/java/com/imageworks/spcue/test/dao/postgres/WhiteboardDaoTests.java

```diff
@@ -268,17 +268,19 @@ public ActionEntity createAction(FilterEntity f) {
     }
 
     public RenderHost getRenderHost() {
-
+        // Hardcoded value of dispatcher.memory.mem_reserved_min
+        // to avoid having to read opencue.properties on a test setting
```

Comment:
Same recommendation: Use cuebot/src/test/resources/opencue.properties for the value, rather than hardcoding it.